torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -233,18 +245,27 @@ RegisterOperators reg({
         [](Node* node) {
           size_t num_inputs = node->inputs().size();
           return [num_inputs](Stack& stack) {
-            bool first = true;
-            for (const IValue& i : last(stack, num_inputs)) {
-              if (!first)
-                std::cout << " ";
-              first = false;
-              std::cout << i;
-            }
+            printListToStream(std::cout, " ", last(stack, num_inputs));
             drop(stack, num_inputs);
             std::cout << std::endl;
             return 0;
           };
         }),
+    Operator(
+        prim::RaiseException,
```

Comment:
I think this should just raise an exception and take a single string argument. String formatting should be done externally to this op.